internal/ir/xds.go

```diff
@@ -694,10 +694,22 @@ type GlobalRateLimit struct {
 type RateLimitRule struct {
 	// HeaderMatches define the match conditions on the request headers for this route.
 	HeaderMatches []*StringMatch
+	// IPMatch define the match conditions on the source IP for this route.
+	IPMatch *IPMatch
```

Comment:
why not list ?

Reply:
https://github.com/envoyproxy/gateway/blob/9c4a15053ef0c645591795f4bbf597e2411cc81f/api/v1alpha1/ratelimitfilter_types.go#L105

you want to combine rules here?

Reply:
I think this is non needed, my bad